### ISSUE TITLE
QuickSchedule button does not appear after install

### ISSUE THREAD

**Issue by sacnayak** [nylas/N1#477]
My use case was:-
- New download of Nylas N1
- Go to Nylas N1 quick schedule repo, get the Quick Schedule plugin.
- Install it
- Double check plugins in the Nylas client if plugin is enabled by default
- Compose message. *Button does not appear*.

However, if:
- Go to plugins, toggle disable/enable twice.
- Check again, *it works*.

**Comment by bengotow** [comment]
Hey @sacnayak — thanks for reporting this. Can you see if the plugin starts working if you open a composer inline (in the main window), or more than two pop-out composers? I think this may be the result of some caching we're doing—new plugins may not appear in composer windows until a few composers have been opened and closed. If you can verify, I can get a fix planned out.

**Comment by sacnayak** [comment]
Hi @bengotow,

I tried replicating the original issue ie. *Quick Schedule button not popping up immediately on install*:
- I uninstalled the plugin
- Reinstalled it
It is not replicable now. 

However, on click of the Quick Schedule button, I get an empty 'Electron' popup as attached. Tried with multiple windows open and within the main window too. Refer screenshots.

Two points:
- Maybe this ticket #477 is replicable on fresh installs only. I am not able to replicate this the second time onwards. Is there a cache I need to clear?
- The empty popup might be another issue altogether and maybe needs another ticket.

<img width="1134" alt="quick schedule within main window" src="https://cloud.githubusercontent.com/assets/6748115/11412537/0c97b160-93ac-11e5-82ca-63404c522c18.png">
<img width="1083" alt="multiple popup windows" src="https://cloud.githubusercontent.com/assets/6748115/11412538/0c9a604a-93ac-11e5-8717-18c9696e74f8.png">
